adoc/deployment.adoc

```diff
@@ -24,33 +24,46 @@ To obtain the credentials from {scca}, follow these steps:
 . Select `Proxies` in the top menu. The credentials are displayed in the top right corner.
 . To see the password, click the 'eye' icon.
 . Create a `password.txt` file containing the obtained password.
+[source]
+  $ head -1 ./password.txt | helm registry login registry.suse.com \
+  --username <PRIVATE_REGISTRY_USERNAME> --password-stdin
+. Create a namespace for {sregistry}.
+[source]
+$ kubectl create namespace <PRIVATE_REGISTRY_NAMESPACE>
 . Store the mirroring credentials retrieved from {scca} as {kube} secrets by running the following command:
 [source]
   $ kubectl create secret docker-registry suse-registry \
-  --docker-server=registry.suse.com --docker-username=<SUSE_REGISTRY_USERNAME> \
+  --namespace <PRIVATE_REGISTRY_NAMESPACE> \
+  --docker-server=registry.suse.com \
+  --docker-username=<PRIVATE_REGISTRY_USERNAME> \
   --docker-password=$(head -1 ./password.txt)
+. Optionally, to use TLS encrypted communication, create a TLS secret from your private key and certificate files.
+[source]
+  $ kubectl create secret tls suse-registry-tls \
+  --namespace <PRIVATE_REGISTRY_NAMESPACE> \
+  --cert=<CERTIFICATE>.pem \
+  --key=<PRIVATE_KEY>.pem
 
 [#pr-deployment-installing-and-running]
 === Installing and running {productnameshort} using {helm}
 The following procedure describes how to install {productnameshort} using {helm}.
 Replace `<RELEASE_NAME>` with your custom release name for the {helm} chart deployment.
 
 . Log in to {sregistry} using the obtained {scca} mirroring credentials.
-[source]
```

Comment:
Why did we remove this section? It's useful! Otherwise the step 1 "Log in to SUSE Registry" could be misunderstood.